react/features/base/participants/middleware.ts

```diff
@@ -188,11 +186,11 @@ MiddlewareRegistry.register(store => next => action => {
         const { deploymentInfo } = state['features/base/config'];
 
         // if there userRegion set let's use it for the local participant
-        if (deploymentInfo && deploymentInfo.userRegion) {
+        if (deploymentInfo?.userRegion) {
             const localId = getLocalParticipant(state)?.id;
 
             store.dispatch(participantUpdated({
-                id: localId,
+                id: localId ?? '',
```

Comment:
why not checking for localId in the if above?

Reply:
`localId` is declared inside the if. I'll add another if